pages/sign.tsx

```diff
@@ -1,5 +1,22 @@
+import MUIButton from '@components/MUIButton'
+
 const sign = () => {
-  return <div>로그인 페이지</div>
+  const onButtonClick = () => {
+    alert('안녕')
+  }
+
+  return (
+    <>
+      <MUIButton
+        onClick={onButtonClick}
+        color={'white'}
+        height={'90px'}
+        width={'80%'}
+        text={'테스트버튼'}
+        border-radius={'50px'}
+        background-color={'red'}></MUIButton>
```

Comment:
버튼 태그는 닫는 태그가 있습니다! 역시 민수님
그렇다면!!? text prop을 받을 필요가 있을까요?? 저희에게 익숙했던 children을 사용하는것은 어떨까요??
사실상 버튼안에 children이 들어가는 경우가 없다면 닫는 태그로 표현해도 좋을 것 같습니다.
```suggestion
      <MUIButton
        onClick={onButtonClick}
        color={'white'}
        height={'90px'}
        width={'80%'}
        border-radius={'50px'}
        background-color={'red'}>테스트버튼</MUIButton>
```